config/nvim/lua/lazy-config.lua

```diff
@@ -1,38 +1,19 @@
 return {
 	require("lazy").setup({
-		require("kickstart.plugins.startup"),
-		require("kickstart.plugins.vim-sleuth"),
-		require("kickstart.plugins.comment"),
-		require("kickstart.plugins.gitsigns"),
-		require("kickstart.plugins.telescope"),
-		require("kickstart.plugins.lspconfig"),
-		require("kickstart.plugins.conform"),
-		require("kickstart.plugins.autocomplete"),
-		require("kickstart.plugins.theme"),
-		require("kickstart.plugins.todo"),
-		require("kickstart.plugins.mini"),
-		require("kickstart.plugins.tree-sitter"),
-		require("kickstart.plugins.vim-helm"),
-	}, {
-		ui = {
-			-- If you have a Nerd Font, set icons to an empty table which will use the
-			-- default lazy.nvim defined Nerd Font icons otherwise define a unicode icons table
-			-- icons = vim.g.have_nerd_font and {} or {
-			icons = {
-				cmd = "⌘",
-				config = "🛠",
-				event = "📅",
-				ft = "📂",
-				init = "⚙",
-				keys = "🗝",
-				plugin = "🔌",
-				runtime = "💻",
-				require = "🌙",
-				source = "📄",
-				start = "🚀",
-				task = "📌",
-				lazy = "💤 ",
-			},
-		},
-	}),
+		require("lazy.autocomplete"),
+		require("lazy.conform"),
+		require("lazy.comment"),
+		require("lazy.oil"),
+		require("lazy.indent_line"),
+		require("lazy.gitsigns"),
+		require("lazy.lspconfig"),
+		require("lazy.mini"),
+		require("lazy.startup"),
+		require("lazy.telescope"),
+		require("lazy.theme"),
+		require("lazy.todo"),
+		require("lazy.tree-sitter"),
+		require("lazy.vim-helm"),
+		require("lazy.vim-sleuth"),
```

Comment:
You should be able to point to the directly with directory 
https://github.com/wd60622/dotfiles/blob/6539f2b8697ebb62eb1afc19fdd73fc123dd334a/nvim/init.lua#L21-L22

Reply:
Thank you! You wouldn't believe the amount of times I forgot just one or two.

Reply:
just has to be within a `lua` directory!